Libplanet/ByteArrayExtension.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Diagnostics.Contracts;
+
+namespace Libplanet
+{
+    /// <summary>
+    /// This extension class enables some convenient methods to deal with byte array.
+    /// </summary>
+    /// <seealso cref="Address"/>
+    public static class ByteArrayExtension
```

Comment:
How about renaming it to `ByteArrayExtensions`?   The convention seems to prefer plural over singular.

Reply:
Renamed every extension class to plural.